### ISSUE TITLE
Moving the project out of the openSUSE organization 

### ISSUE THREAD

**Issue by apinter** [openSUSE/openSUSE-docs-revamped-temp#81]
For all who still happened to contribute here,

Considering the past issues, incidents around this project, and individuals conflict with the community, along with the rising levels of apathy the Docs is going through - likely due to above mentioned issues - I would suggest the move of the project out of the openSUSE Gitlab organization. Doing that would truly achieve what is already stated in the project: it is a community driven and unofficial project.
 
In it's current state it is more of a source of issues for users (unmaintained docs) than help, and user issues are trickling down to the numerous support channels. The #80 is a great example. Unmaintained doc conflicting with devs and users, and frankly I find this quite simply unacceptable as it introduces an additional, unneeded burden to other volunteers.

IMO we should either move this project out of the openSUSE org or archive it as is.

**Comment by why-not-try-calmer** [comment]
A handful of clarifications:
- the "incidents" you mention have been dealt with by the previous iteration of the Board, and the sentence associated with it, served;
- before Stefan had his account deleted and left SUSE, we were working on a PR to finish migrating these docs to doc.opensuse.org; I am not sure what happened to him but I am confident that this move can carry on with his successor (see #78)
- in the mean time I've become a father of one and I confess that has left me with less time for curating these docs as I had anticipated (so on that front I completely agree: we need more maintainers!)
- I am currently "training" / showing the ropes to another openSUSE contributor (@SingletonW) which has approached me a few months ago and has shown a desire to become a maintainer in the long run -- to be seen how this develops); 
- if I look at the platforms (Reddit and Telegram) I see a good number of folks mentioning, recommending and otherwise talking positively of this documentation (hell, I even remember a Destination Linux episode where they talked about it!); so even factoring out "folks mislead by microOS documentation" (PR now merged) that leaves out a number of people who value the work done here by openSUSE contributors.

For these reasons, I don't think this would be a very good idea. I believe this documentation has already repaid tenfold for the work that went into it and I don't think there's any reason to doubt it will help more openSUSE users in the future.

**Comment by apinter** [comment]
Non of what happened months ago changes the facts: This project is unmaintained and other projects seem to fill the gaps caused by this one which is absolutely unfair. 
Even the PR you're referencing is over 2 months old. This is the reason why I recommend to move it out of the openSUSE organization or work directly with the SUSE docs team.
If what you're saying is indeed the case then it is also pointless to keep the 2 projects alive, and just merge it under the SUSE one.

For the record I'm not asking you about this as a Board member, but the person who actually started this entire project. Either get in touch with the developers, doc maintainers and bring the docs up-to-date in a fashion that is not causing further issues, and clashes or move it to your care elsewhere. Simple really and don't seem to see an issue with it as it doesn't hurt you, but also helps the other projects, maintainers, devs, and people supporting users.

**Comment by why-not-try-calmer** [comment]
> Either get in touch with the developers, doc maintainers and bring the docs up-to-date in a fashion that is not causing further issues, and clashes or move it to your care elsewhere. Simple really and don't seem to see an issue with it as it doesn't hurt you, but also helps the other projects, maintainers, devs, and people supporting users.

Cool. Will do!

**Comment by apinter** [comment]
> Cool. Will do! Any particular suggestion on what is outdated and could use an update?

I'm not here to tell you what to do, I'm telling you what will happen if you continue to do nothing ;)

**Comment by apinter** [comment]
This is far from a done conversation and should be very much on the table, not sure why did you close it.

**Comment by SingletonW** [comment]
You're right. I don't understand what you're trying to achieve. For sure, this project could do more to make itself visible and accessible. This includes the "decision process" part. We are very much aware of this. Also that's the best way I can think of to improve the commits frequency - which, contrary to the narrative you're pushing, is similar to other projects under /openSUSE/ with around the same number of contributors.
As you can see, there are people that come here and help -just look at the recent Git history. These people are community members like you and they "vote with their feet". By helping this project. By submitting PRs. By sharing links pointing to these contents. They are doing their part, and that's freaking amazing.
I look at the Git history. I see that almost all PRs have been dealt with swiftly. I see that almost all were accepted. So I think this project _is_ also responsive to community inputs. But if someone comes here to throw a bunch of threats, demeaning judgments and sarcasms in a patronizing tone, for sure this is not going to lend credence to the notion that they just want a "discussion" or to just "make a suggestion".

As I said, we should absolutely do more to become more visible and responsive to the needs of the community. This is actually my main incentive for trying to become a maintainer here, so I am not taking this lightly. There is certainly something we should do on the community's social platforms, to get constructive criticisms. But the all guns blazing stunts are alien to this effort I think.

**Comment by apinter** [comment]
I think Adrien understands the problem better since he has been made aware for a while now so you should probably talk with them to have a clear understanding, but going to make a PR which addresses this issue that might help clear the picture.

> As I said, we should absolutely do more to become more visible and responsive to the needs of the community

Oh the community is aware... Happy to drop by and address this issue during the next docs meeting if the above remains unclear for you.
